src/Features/VisualBasic/Portable/Diagnostics/Analyzers/VisualBasicNamingStyleDiagnosticAnalyzer.vb

```diff
@@ -1,11 +1,21 @@
 ﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+Imports System.Collections.Immutable
 Imports Microsoft.CodeAnalysis.Diagnostics
 Imports Microsoft.CodeAnalysis.Diagnostics.Analyzers.NamingStyles
 
 Namespace Microsoft.CodeAnalysis.VisualBasic.Diagnostics.Analyzers
     <DiagnosticAnalyzer(LanguageNames.VisualBasic)>
-    Friend Class VisualBasicNamingStyleDiagnosticAnalyzer
-        Inherits NamingStyleDiagnosticAnalyzerBase
+    Friend NotInheritable Class VisualBasicNamingStyleDiagnosticAnalyzer
+        Inherits NamingStyleDiagnosticAnalyzerBase(Of SyntaxKind)
+
+        Private Shared _supportedSyntaxKinds As ImmutableArray(Of SyntaxKind) =
+            ImmutableArray.Create(SyntaxKind.ModifiedIdentifier)
```

Comment:
static should be s_

Reply:
actually maybe this shouldn't be static but should be readonly - that would be the equivalent of the c# version

Reply:
actually this should just be a readonly auto-property - I didn't know VB had them

Reply:
resolved in 926a46847bdb00abb502a3088863ebd3fcfa04ca